src/panels/Panel_Output.tsx

```diff
@@ -69,7 +69,7 @@ export const SideOutputListUI = observer(function SideOutputListUI_(p: {}) {
     const step = st.focusedStepL
     const size = st.historySizeStr
     return (
-        <div tw={'flex flex-wrap gap-0.5 p-1 overflow-auto flex-shrink-0 bg-base-300 items-center'}>
+        <div tw={'flex flex-wrap gap-0.5 p-1 overflow-auto flex-shrink-0 bg-base-300 items-center max-h-[50%]'}>
```

Comment:
For workflows with many results, this pane would take over the whole preview size, this ensures is stays less than half